src/executor/query.rs

```diff
@@ -198,7 +200,7 @@ macro_rules! try_getable_mysql {
                     #[allow(unused_variables)]
                     QueryResultRow::Mock(row) => row.try_get(_column.as_str()).map_err(|e| {
                         debug_print!("{:#?}", e.to_string());
-                        TryGetError::Null
+                        TryGetError::Null(_column)
```

Comment:
Since we are using the variable now, we should remove the `_` prefix

Reply:
I fixed in https://github.com/SeaQL/sea-orm/pull/853/commits/3ee0662d123abf7af626903b19b1d833f3fb9403